### PR TITLE
S2 task1

### PR DESCRIPTION
**Pull request by richardjgabb**
Sort feature added

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -45,15 +39,20 @@
         <div class="passportPage">
         <div class="passportDiv">
             <?php
-            if (($_GET['sortBy'] === 'date') || ($_GET['sortBy'] === 'country') || ($_GET['sortBy'] === 'rating')) {
+            if (!isset($_GET['sortBy'])) {
```
Comment (JBHarrison5) [inline index.php:42]: could combine into one if statement?

File: index.php [hunk 2]
```diff
@@ -45,15 +39,20 @@
...
+                $sorter = 'date';
```
Comment (JBHarrison5) [inline index.php:43]: hardcoded?

File: index.php [hunk 3]
```diff
@@ -45,15 +39,20 @@
...
+                $query->execute();
+                $database = $query->fetchAll();
+            } elseif (($_GET['sortBy'] === 'date') || ($_GET['sortBy'] === 'country') || ($_GET['sortBy'] === 'rating')) {
                 $sorter = $_GET['sortBy'];
                 $query = $db->prepare("SELECT `airport`,`country`, `date`, `image`, `rating`, `review`
                 FROM `stamps`
                 ORDER BY `" . $sorter . "`;");
                 $query->execute();
                 $database = $query->fetchAll();
-            } else {
```
Comment (JBHarrison5) [inline index.php]: think most of this can be combined into just an if-else to save repeating the code so much

**Review by matt-ioa — APPROVED** [review]
Good job!

**Pull request merged**